Xcode-Configurations/Release.xcconfig

```diff
@@ -1,6 +1,6 @@
 #include "Base"
 
-GCC_PREPROCESSOR_DEFINITIONS = NS_BLOCK_ASSERTIONS
-WARNING_CFLAGS = -Wall -Wno-nonnull -Wno-property-attribute-mismatch -Wno-unguarded-availability
-
+VALIDATE_PRODUCT = YES
+ENABLE_NS_ASSERTIONS = NO
 GCC_TREAT_WARNINGS_AS_ERRORS = YES
+WARNING_CFLAGS = -Wall -Wno-nonnull -Wno-property-attribute-mismatch -Wno-auto-import
```

Comment:
What kind of warnings were we hitting with `-Wauto-import` that we want to avoid seeing?

Reply:
This arose from reconciling the sample project configs with the shared config. `CLANG_ENABLE_MODULES = YES` is the default in newer Xcode projects (including all of the sample projects) and required in even newer ones. When you `-Weverything` with modules enabled, you get spammed with tens of thousands of, for example: `treating #import as an import of module 'GitUpKit'`, from `-wauto-import`. Adopting module imports in the hundreds of headers could be taken on separattely, but it was definitely not my intent to do so in this PR or any time soon.